.flake8

```diff
@@ -12,41 +12,14 @@ exclude =
     tutorials/tagger/**
 
 per-file-ignores =
-    # __init__.py files are allowed to have unused imports and lines-too-long
-    allennlp/__init__.py:F401
-    allennlp/**/__init__.py:F401,E501
+    # __init__.py files are allowed to have unused imports
+    */__init__.py:F401
+    */**/**/__init__.py:F401
```

Comment:
Apparently this works. If you can come up with a way to grab all with one line, that'd be cool.

Reply:
Please leave the `lines-too-long`; black will catch anything that's fixable, so we don't need flake to complain at us for these files.  Even if there aren't any files where it matters now, it'd be annoying to have this come up later.